src/ml/neural_net/mps_layer_conv_padding.m

```diff
@@ -0,0 +1,65 @@
+#import <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation TCMPSConvolutionPadding
+
++ (BOOL) supportsSecureCoding {
+  return YES;
+}
+
+- (instancetype) initWithParams:(NSUInteger)paddingWidth
+                  paddingHeight:(NSUInteger)paddingHeight
+                    strideWidth:(NSUInteger)strideWidth
+                   strideHeight:(NSUInteger)strideHeight {
+  
+  self = [self init];
+
+  if (self) {
+    _paddingWidth = paddingWidth;
+    _paddingHeight = paddingHeight;
+    _strideWidth = strideWidth;
+    _strideHeight = strideHeight;
+  }
+
+  return self;
+}
+
+- (nullable instancetype)initWithCoder:(NSCoder *)coder {
```

Comment:
Does this need to be overridden? Can we just allow the superclass `NSObject` to handle it?

Reply:
Unfortunately the same issue:

```
/Users/abhishekpratapa/Desktop/turicreate/src/ml/neural_net/mps_layer_conv_padding.m:27:10: error: no visible @interface for 'NSObject' declares the selector 'initWithCoder:'
```